app/src/main/res/xml/preferences.xml

```diff
@@ -16,29 +16,5 @@
   -->
 
```

Comment:
Maybe we should add a comment here that says something like "Preferences are always added dynamically from Settings Activity." so this blank file doesn't confuse anybody.